tests/test_client.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from PIL import Image
+
+from stability_sdk import client
+import stability_sdk.interfaces.gooseai.generation.generation_pb2 as generation
+import stability_sdk.interfaces.gooseai.generation.generation_pb2_grpc as generation_grpc
+
+def test_client_import():
+    from stability_sdk import client
+    assert True
+
+def test_StabilityInference_init():
+    class_instance = client.StabilityInference(key='thisIsNotARealKey')
+    assert True
+
+def test_StabilityInference_init_nokey_error():
+    try:
+        class_instance = client.StabilityInference()
+        assert False
+    except ValueError:
+        assert True
+
+def test_StabilityInference_init_nokey_insecure_host():
+    class_instance = client.StabilityInference(host='foo.bar.baz')
+    assert True
+
+def test_image_to_prompt():
+    im = Image.new('RGB',(1,1))
+    prompt = client.image_to_prompt(im, init=False, mask=False)
+    assert isinstance(prompt, generation.Prompt)
+
+def test_image_to_prompt_init():
+    im = Image.new('RGB',(1,1))
+    prompt = client.image_to_prompt(im, init=True, mask=False)
+    assert isinstance(prompt, generation.Prompt)
+
+def test_image_to_prompt_mask():
+    im = Image.new('RGB',(1,1))
+    prompt = client.image_to_prompt(im, init=False, mask=True)
+    assert isinstance(prompt, generation.Prompt)
+
+def test_image_to_prompt_init_mask():
+    im = Image.new('RGB',(1,1))
+    try:
+        prompt = client.image_to_prompt(im, init=True, mask=True)
+        assert False
+    except ValueError:
+        assert True
```

Comment:
very minor nit:  In cases where we're expecting errors, it may be worthwhile to verify that the exception message is the one we expect as well.  It will make the test more specific as constraints evolve into the future.

Reply:
i'll merge it for now and do a separate PR to add those test improvements, thanks for the feedback!